.config/dotnet-tools.json

```diff
@@ -1,41 +1,34 @@
-{
-  "version": 1,
-  "isRoot": true,
-  "tools": {
-    "paket": {
-      "version": "8.0.3",
-      "commands": [
-        "paket"
-      ],
-      "rollForward": false
-    },
-    "dotnet-reportgenerator-globaltool": {
-      "version": "5.3.8",
-      "commands": [
-        "reportgenerator"
-      ],
-      "rollForward": false
-    },
-    "fantomas": {

```

Comment:
@TheAngryByrd Was the removal of fantomas because of the nuget bug in 9.0.100 or on purpose? Just curious, no hard feelings if it was on purpose.

Reply:
almost certainly the bug - it can come back now!